sdk/__init__.py

```diff
@@ -76,6 +80,12 @@ def create_batch(
         if device_type != DeviceType.QPU:
             req.update({"emulator": device_type})
 
+        # The configuration field is only added in the case
+        # it's requested
+        if configuration:
+            if isinstance(configuration, Configuration):
+                configuration = configuration.to_dict()
+            req.update({"configuration": json.dumps(configuration)})
```

Comment:
Dump to string won't be necessary anymore if the changes from str -> JSON is made on the public_apis side

Reply:
Done